docs/training.md

```diff
@@ -1,186 +1,259 @@
+# Training and comparing models
+
+Before you can make predictions, you need to train a model using known data.
 
 ## What is `SupervisedModelTrainer`?
 
--   This class let's you create and compare machine learning models on diverse
+- This class lets you train and compare machine learning models on diverse
     datasets.
--   You can do both **classification** (for example, predict Y/N) as well as
-    **regression** (for example, predict a numeric field).
--   To jump straight to an example notebook, see
-    [here](https://github.com/HealthCatalystSLC/healthcareai-py/blob/master/notebooks/Example1.ipynb)
+- You can do both **classification** (for example, predicting Y/N) as well as
+    **regression** (for example, predicting a numeric value).
 
 ## Am I ready for model creation?
 
-Maybe. It'll help if you follow these guidelines:
+To build a model, please follow these guidelines for setting up your training data:
+
+- Don't use `0` or `1` for the dependent variable when doing binary classification. Use `Y`/`N` instead. The `IIF` function in T-SQL is an easy way to convert your target variables.
+- Don't pull in test data in this step. In other words, we just pull in those rows where the target (predicted column) has a value already.
+- Feature engineering is always a good idea. Check out our [blog](http://healthcare.ai/blog) for ideas.
+
+## Step 1: Load training data
 
-> -   Don't use `0` or `1` for the dependent variable when doing
->     classification. Use `Y`/`N` instead. The IIF function in T-SQL may
->     help here.
-> -   Don't pull in test data in this step. In other words, we just pull
->     in those rows where the target (ie, predicted column has a value
->     already).
+### MSSQL
 
-Feature engineering is always a good idea. Check out our [blog](http://healthcare.ai/blog] for ideas.
+```python
+server = 'localhost'
+database = 'SAM'
+query = """SELECT *
+            FROM [SAM].[dbo].[DiabetesClincialSampleData]
+            -- In this step, just grab rows that have a target
+            WHERE ThirtyDayReadmitFLG is not null"""
+engine = hcai_db.build_mssql_engine(server=server, database=database)
+dataframe = pd.read_sql(query, engine)
 
-## Step 1: Pull in the data
+# Handle missing data (if needed)
+dataframe.replace(['None'], [None], inplace=True)
+```
 
-For SQL:
+### CSV
 
 ```python
-import pyodbc
-cnxn = pyodbc.connect("""SERVER=localhost;
-                        DRIVER={SQL Server Native Client 11.0};
-                        Trusted_Connection=yes;
-                        autocommit=True""")
+dataframe = pd.read_csv('healthcareai/tests/fixtures/DiabetesClinicalSampleData.csv', na_values=['None'])
+```
+
+
+## Step 2: Set up a Trainer
 
- df = pd.read_sql(
-     sql="""SELECT *
-            FROM [SAM].[dbo].[HCPyDiabetesClinical]""",
-     con=cnxn)
+The `SupervisedModelTrainer` class helps you train models. It cleans and prepares the data before model creation. To set up a trainer you'll need these arguments:
```

Comment:
Something like, "It cleans and preps the data and also assigns parameters to be used in model creation." Or maybe "It also assigns parameters to be used in the model creation."